juniper_official/Linecard/pipe-stats.rule

```diff
@@ -0,0 +1,224 @@
+healthbot {
+    topic external {
```

Comment:
Change topic name based on as per the domain.

Reply:
Done